src/ur5_simulation/ur5model.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed May 12 20:48:22 2021
+
+@author: Rafael Queiroz
+"""
+
+import pandas as pd
+import numpy as np
+from math import cos, sin
+
+class UR5:
+        
+    def __init__(self, d1=0.089159, a2=-0.42500, a3=-0.39225, d4=0.1091,
+                 d5=0.09465, d6=0.0823):
+        self.d1 = d1
+        self.a2 = a2
+        self.a3 = a3
+        self.d4 = d4
+        self.d5 = d5
+        self.d6 = d6 
+        self.t1, self.t2, self.t3, self.t4, self.t5, self.t6 = (0,0,0,0,0,0)
+           
+    @property
+    def data(self):
+        return np.array([[self.t1, self.d1, 0, 0],
+                         [self.t2, 0, 0, 90],
+                         [self.t3, 0, self.a2, 0],
+                         [self.t4, self.d4, self.a3, 0],
+                         [self.t5, self.d5, 0, 90],
+                         [self.t6, self.d6, 0, -90]])
```

Comment:
This is being currently done by the function `def fkine(self, q=[0, 0, 0, 0, 0, 0]):`.  In `    @property
    def data(self):`, the first column is a set of attributes, that is supposed to be changed by `def fkine(self, q=[0, 0, 0, 0, 0, 0]):`. However, it can be condensed in a single function, if desired.